### DISCUSSION TITLE
New theming engine

### DISCUSSION THREAD

**Discussion by igordutra** [primefaces/community#1293] [Figma UI Kit] [answered]
Can you provide an update on the new theming engine? Is it still on track for Q1? 

I'm planning to use primevue for a new project (with tailwind presets) and we need to apply a new theme the library and components. Ideally I'd like to work on the new engine file otherwise all my work will be lost. 

Also, is it possible to get involved and contribute to the Figma UI Kit?

Many thanks
Igor

**Comment by onursenture** [comment, marked as answer]
Hi, the first beta for the new styled mode is scheduled for the second week of March, with the final release due at the end of March. 🤞 I'm uncertain about the changes for the unstyled mode (Tailwind) and its roadmap. Perhaps @atakantepe could provide some information on this subject.

Regarding the Figma UI Kit, it's being maintained internally by the PrimeTek design team, so we're unable to accept contributions at this time. However, we're always open to feedback and appreciate your involvement. Thank you.
